### DISCUSSION TITLE
How to track SQL errors

### DISCUSSION THREAD

**Discussion by lozdown** [sqlpage/SQLPage#191] [Q&A]
How can I see SQL execution errors returned from the database  back end (in my case postgresql).

**Comment by lovasoa** [comment]
They should appear directly on the web page that contains the error, and in SQLPage 's logs. 

Can you share an example with a screenshot of your problem?

**Comment by lozdown** [comment]
Thanks 
Where are the SQLPage logs located?

**Reply by lovasoa** [reply]
The logs are written on the standard error output (stderr). If you started SQLPage in a terminal, you can see them in the terminal window. 

Can you please provide an example of the sql file you wrote and a screenshot of the resulting webpage? The error should be displayed on the page.
